This patch is needed to convert BSD-style section numbers to Solaris-style
section numbers.  It has not been fed back to the upstream community.  It
may need to be tweaked slightly for future releases.

--- libarchive/libarchive/archive_entry_linkify.3.orig	Mon Mar 26 19:49:00 2012
+++ libarchive/libarchive/archive_entry_linkify.3	Fri Nov  9 12:38:22 2012
@@ -23,7 +23,7 @@
 .\" SUCH DAMAGE.
 .\"
 .Dd February 2, 2012
-.Dt ARCHIVE_ENTRY_LINKIFY 3
+.Dt ARCHIVE_ENTRY_LINKIFY 3ARCHIVE
 .Os
 .Sh NAME
 .Nm archive_entry_linkresolver ,
@@ -91,7 +91,7 @@
 .Fn archive_entry_linkresolver_set_strategy
 function selects the optimal hardlink strategy for the given format.
 The format code can be obtained from
-.Xr archive_format 3 .
+.Xr archive_format 3ARCHIVE .
 The function can be called more than once, but it is recommended to
 flush all deferred entries first.
 .Pp
@@ -218,7 +218,7 @@
 returns
 .Dv NULL
 on
-.Xr malloc 3
+.Xr malloc 3C
 failures.
 .Sh SEE ALSO
-.Xr archive_entry 3
+.Xr archive_entry 3ARCHIVE
